lib/utils/OcrIngredientsField.dart

```diff
@@ -0,0 +1,21 @@
+enum OcrIngredientsField {
```

Comment:
I think we should remove this enum, there are 200 languages, we cannot and should not list them all.

So instead of having the user specify "INGREDIENTS_DE" etc. the user should only specify the language code (lc) and set it to the string "de".